src/test/scala/io/iohk/ethereum/vm/Generators.scala

```diff
@@ -48,17 +49,18 @@ object Generators extends ObjectGenerators {
     memGen: Gen[Memory] = getMemoryGen(),
     storageGen: Gen[Storage] = getStorageGen(),
     codeGen: Gen[ByteString] = getByteStringGen(0, 0),
-    callDataGen: Gen[ByteString] = getByteStringGen(0, 0),
-    callValueGen: Gen[ByteString] = getByteStringGen(0, 32)
-  ) : Gen[ProgramState] =
+    inputDataGen: Gen[ByteString] = getByteStringGen(0, 0),
+    valueGen: Gen[BigInt] = getBigIntGen(0)
```

Comment:
Ahh, just noticed the type changed, but still I think we should use `2^256 - 1` as the max value.